src/sentry/release_health/duplex.py

```diff
@@ -446,7 +446,16 @@ def __init__(
     ):
         self.sessions = SessionsReleaseHealthBackend()
         self.metrics = MetricsReleaseHealthBackend()
-        self.metrics_start = metrics_start
+        self.metrics_start = max(
+            metrics_start,
+            # The sessions backend never returns data beyond 90 days, so any
+            # query beyond 90 days will return truncated results.
+            # We assume that the release health duplex backend is sufficiently
+            # often reinstantiated, at least once per day, not only due to
+            # deploys but also because uwsgi/celery are routinely restarting
+            # processes
+            datetime.now(timezone.utc) - timedelta(days=89),
```

Comment:
Why 89 not 90?

Reply:
i didn't want to think about off-by-one errors too much and wanted to err on the side of rather not comparing so we don't have to deal with as many potential false-positives. there's not a lot of mismatches that don't already appear at the 30d range, i assume